src/components/experiments/single-view/overview/VariationsTable.tsx

```diff
@@ -48,12 +48,14 @@ function assignmentHref(variationName: string, experimentName: string) {
   nameSchema.validateSync(variationName)
   return `javascript:(async () => {
     const token = JSON.parse(localStorage.getItem('experiments_auth_info'));
+    const anonId = document.cookie.match('(^|;)\\\\s*tk_ai\\\\s*=\\\\s*([^;]+)')?.pop() || '';
```

Comment:
I recommend keeping all information necessary in the `body` if we are sending a body rather than mixing between query params and body. One way to do that is to ensure the correct encoding of anon_id in this code:

```suggestion
    const anonId = decodeURIComponent(document.cookie.match('(^|;)\\\\s*tk_ai\\\\s*=\\\\s*([^;]+)')?.pop() || '');
```

That way we are making the encoding explicit instead of relying on PHPs automatic GET/cookie decoding.

Reply:
Sounds good. That was my first idea, but I figured that passing the `anon_id` as a query parameter matches the assignment GET.